riff-raff/app/controllers/DeployController.scala

```diff
@@ -184,8 +183,10 @@ class DeployController(
       .filter(p => p.number.contains(term) || p.branchName.contains(term))
       .map { build =>
         val label =
-          "%s [%s] (%s)" format (build.number, build.branchName, shortFormat
-            .print(build.startTime))
+          "%s [%s] (%s) (%s)" format (build.number, build.branchName, shortFormat
+            .print(build.startTime), VCSInfo
+            .normalise(build.vcsURL)
+            .getOrElse(build.vcsURL))
```

Comment:
`scalafmt` making things super readable again 😅. Might be worth pulling this out to a `val`?

```scala
val repository = VCSInfo.normalise(build.vcsURL).getOrElse(build.vcsURL)
val label = ...
```